students/hirot_directory/session04/mailroom2.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+
+donor_names = ["William Gates III", "Mark Zuckerberg", "Jeff Bezos", "Paul Allen"]
+donations = [[200, 500], [2000, 3000], [4000, 5000], [100, 150]]
+donor_dict = dict(zip(donor_names, donations))
+
+def thank_you():
+
```

Comment:
This is a little clunky, and doesn't actually ever print a letter. What you really want here is another menu where you can either enter a name/donation (and print a letter!), or print 'list' to get all of the names (which returns to this same menu) or exits to the main menu).